site/_data/docs/navigation.yml

```diff
@@ -258,12 +258,18 @@
         path: /react/scheduler/docs/guides/getting-started
   - title: API Reference
     items:
+      - title: Scheduler
+        path: /react/scheduler/docs/reference/scheduler
       - title: AppointmentForm
         path: /react/scheduler/docs/reference/appointment-form
       - title: ViewState
         path: /react/scheduler/docs/reference/view-state
+      - title: DayView
+        path: /react/scheduler/docs/reference/day-view
       - title: WeekView
         path: /react/scheduler/docs/reference/week-view
+      - title: MonthView
+        path: /react/scheduler/docs/reference/month-view
```

Comment:
Can we use alphabet order here? Because I think it's good for UX

Reply:
This order doesn't make sense. We [sort items](https://github.com/DevExpress/devextreme-reactive/blob/master/site/_layouts/docs.html#L31) in the docs layout.